go/libkb/errors.go

```diff
@@ -1399,6 +1399,12 @@ func (e UnknownStreamError) Error() string {
 	return "unknown stream format"
 }
 
+type UnicodeUnsupportedError struct{}
+
+func (e UnicodeUnsupportedError) Error() string {
+	return "unicode not supported"
```

Comment:
Could we change this to "UTF-16 not supported"? Although saltpack only expects characters in the ASCII-compatible range, saltpack files are valid UTF-8.